.github/workflows/release.yml

```diff
@@ -29,9 +29,10 @@ jobs:
           helm repo add workflows https://parodos.dev/serverless-workflows-config
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.6.0
+        uses: masayag/chart-releaser-action@mainhelm/chart-releaser-action@v1.6.0
```

Comment:
is it good that we have an action that refers to a personal account?
Can we define it in parados-dev somehow?

Reply:
well, we contributed a PR for the helm chart, but we're waiting for them to even review it.
at the meantime, using a local build that contains exactly the same commit.